libs/design-system/src/component-audit/ComponentAuditTable.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { StoryFn, Meta } from '@storybook/react';
+import { ComponentAuditTable } from './ComponentAuditTable';
+import { css } from '../../styled-system/css';
+
+import scanJson from './component-scans/scan.json';
+
+export default {
+  title: 'ComponentAudit',
+  component: ComponentAuditTable,
+  argTypes: {},
+} as Meta<typeof ComponentAuditTable>;
+
+const TableWrapper = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <div
+      className={css({
+        overflowX: 'auto',
```

Comment:
`overflowY: 'auto'` and we should set a height otherwise the table won't be scrollable

Reply:
Sounds good. To be honest, I spent very little time on this table itself (including the wrapper) since I just asked ClaudeAI to write it for me 😅